challenges/objects-arrays.js

```diff
@@ -138,22 +141,25 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
+  zooAnimals.map(function(name) {
+    lowerCase.push(name.animal_name.toLowerCase());
```

Comment:
I see how you went about this and I have to say a good job. One thing though is when using .map() it loops through the array and pushes back a new array. So if you were to say that lowerCase was assigned to zooAnimals using map, it should give you the array that you want without having to push it because .map is not a .forEach() where all it does is loop through each array. Other than that, good job on getting it done.